coalib/bearlib/aspects/exceptions.py

```diff
@@ -0,0 +1,47 @@
+class AspectTypeError(TypeError):
+    """
+    This error is raised when an object is not an ``aspectclass`` or an
+    instance of ``aspectclass``
+    """
+
+    def __init__(self, item):
+        self.item = item
+        message = ('{} is not an aspectclass or an instance of an '
+                   'aspectclass'.format(repr(self.item)))
+        super().__init__(message)
+
+
+class AspectLookupError(LookupError):
+    """
+    Error that raised when trying to search aspect.
```

Comment:
Error that raised -> Error that is raised ..

or -> Error raised ..

But "No aspect found." is more in the same style as your other docstrings.

Your choice ;-)